tests/resources/install-libhkl.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
-# conda activate hkl
-tar xzf /home/libhkl.tar.gz -C /opt/conda
-echo "VERSION: $(grep VERSION /opt/conda/include/hkl-5/hkl.h)"
+export CONDA_DIR=${CONDA_DIR:-/opt/conda}
+export CONDA_PREFIX=${CONDA_PREFIX:-/opt/conda}
+echo CONDA_DIR="${CONDA_DIR}"
+echo CONDA_PREFIX="${CONDA_PREFIX}"
+
+source $CONDA_DIR/etc/profile.d/conda.sh
+tar xzf /home/libhkl.tar.gz -C $CONDA_PREFIX
```

Comment:
Same note as earlier about missing dash.  It seems to work without the dash for the CLI options.

```suggestion
tar -xzf /home/libhkl.tar.gz -C $CONDA_PREFIX
```

TIL that the dash is optional.